ssl/ssl_local.h

```diff
@@ -1201,13 +1224,21 @@ struct ssl_ctx_st {
     const EVP_MD *ssl_digest_methods[SSL_MD_NUM_IDX];
     size_t ssl_mac_secret_size[SSL_MD_NUM_IDX];
 
+    size_t tls12_sigalgs_len;
     /* Cache of all sigalgs we know and whether they are available or not */
     struct sigalg_lookup_st *sigalg_lookup_cache;
+    /* List of all sigalgs (code points) available, incl. from providers */
+    uint16_t *tls12_sigalgs;
 
     TLS_GROUP_INFO *group_list;
     size_t group_list_len;
     size_t group_list_max_len;
 
+    TLS_SIGALG_INFO *sigalg_list;
+    SSL_CERT_LOOKUP *ssl_cert_info;
```

Comment:
It seems strange to insert this field here, surrounded by other fields all related to `sigalg_list`.

Reply:
Reason for this is that it has the same length. But I'm happy to re-locate as you see fit.

Reply:
Perhaps move it to where other `cert` stuff is declared, e.g. near this line:

````
struct cert_st /* CERT */ *cert;
````

Reply:
I assume you're only suggesting that the `ssl_cert_info` line gets moved, not the `sigalg_list` line, @mattcaswell ?

Reply:
Yes